### ISSUE TITLE
Code Art

### ISSUE THREAD

**Issue by dkoloditch** [codefordenver/projects#19]
A code art project that uses technology to draw attention to a community issue. City of Denver might be interested in hosting this in a public place.

See https://devart.withgoogle.com/ for examples.

Possibly long term.

**Comment by dviramontes** [comment]
**+1**

**Comment by dviramontes** [comment]
It's called:

> ***dont talk about politics***

I don't have it all solidified but with your help maybe we can 
make it more concrete. 

Its basically about how most of don't get to choose to be political
because our existence is politicized from birth.

Have you ever had a friend say, 
***"why you gotta talk about politics man?"*** as if it it
was an option for some people.

With upcoming presidential election there will be a lot of data
specifically twitter data what we can choose to re-interpret 
in a different way.

More to come..

**Comment by eemshi** [comment]
I really like this idea David

A while ago I saw this kickstarter for a project that was exploring a new way to contextualize keywords and hashtags on social media:
https://www.kickstarter.com/projects/violetreport/decolonize-the-internet/?ref=kicktraq

The concept was interesting but maybe too inaccessible, it never got funded... Anyway there could be a simpler (artful) way of combining and contextualizing existing data to paint a landscape of the conversations that are happening on twitter, fb, etc..

**Comment by dviramontes** [comment]
@eemshi yes! exactly and wow what a crazy cool project. Thanks for sharing, looking forward to possibly borrowing ideas from it.
